index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
+<style>
+table, th, td {
+    border: 1px solid black;
+}
+</style>
+</div class="header">
+
+<div id ="nav">
```

Comment:
There's a `<nav>` tag that can be more semantically appropriate: https://www.w3schools.com/tags/tag_nav.asp

Reply:
bootstrap also has a nav class if you are using that framework, in which case it is best to combine `<nav id="nav" class="nav ...">...</nav>`.  Id should not be necessary I think, also nav as an id is not very descriptive, maybe primary-navbar, title-navbar, main-navigation-sidebar or so on.